### PR TITLE
MISUV-8796 - optout completion page content change

### PR DESCRIPTION
**Pull request by TomRafferty**
parent ticket:
['Opt out : 'Completion page' for CY or CY+1 different scenarios' (MISUV-8526)](https://jira.tools.tax.service.gov.uk/browse/MISUV-8526)

work ticket:
['Update view for new content' (MISUV-8796)](https://jira.tools.tax.service.gov.uk/browse/MISUV-8796)

### CODE REVIEW

File: app/views/optOut/ConfirmedOptOut.scala.html [hunk 1]
```diff
@@ -88,6 +78,37 @@ <h2 class="govuk-heading-m">@messages("optout.confirmedOptOut.report-quarterly-n
     </ul>
 }
 
+
+@yourReportingFrequencyBlock = @{
+    val reportingFrequencyUrl = controllers.routes.ReportingFrequencyPageController.show().url
```
Comment (avelx) [inline app/views/optOut/ConfirmedOptOut.scala.html:83]: might be easy to pass this URL as a parameter to corresponding viewModel to reduce dependency etc

File: app/views/optOut/ConfirmedOptOut.scala.html [hunk 2]
```diff
@@ -31,8 +31,7 @@
     govukInsetText : GovukInsetText
 )
 
-@(viewModel: ConfirmedOptOutViewModel, isAgent: Boolean = false)(implicit messages: Messages, user: auth.MtdItUser[_])
-
+@(viewModel: ConfirmedOptOutViewModel, isAgent: Boolean = false, showReportingFrequencyContent: Boolean)(implicit messages: Messages, user: auth.MtdItUser[_])
```
Comment (avelx) [inline app/views/optOut/ConfirmedOptOut.scala.html:34]: we already have isAgent details in user/(in MtdItUser type), so this is a duplicate flag, can be removed;
+ we might need to raise a tech debt to discuss if we need these details across rest of the views

File: app/views/optOut/ConfirmedOptOut.scala.html [hunk 3]
```diff
@@ -88,6 +78,37 @@ <h2 class="govuk-heading-m">@messages("optout.confirmedOptOut.report-quarterly-n
...
+        messageKey = "optout.confirmedOptOut.yourReportingFrequency.text"
+    )
+
+    Html(messages("optout.confirmedOptOut.yourRevisedDeadlines.desc2", reportingFrequencyLink.toString()))
+}
+
+@revisedDeadlinesBlock = {
+    <div id="revised-deadlines" class="govuk-!-margin-bottom-7 govuk-!-margin-top-8">
+        <h2 id="revised-deadlines-heading" class="govuk-heading-m" >@messages("optout.confirmedOptOut.yourRevisedDeadlines.h2")</h2>
+        <p id="revised-deadlines-p1" class="govuk-body" >@messages("optout.confirmedOptOut.yourRevisedDeadlines.desc1", viewModel.startYear, viewModel.endYear, viewModel.nextYear)</p>
```
Comment (Mbrookesy) [inline app/views/optOut/ConfirmedOptOut.scala.html:96]: The section of the text that says `31 January <Year>` should be bold.

Comment (TomRafferty) [reply app/views/optOut/ConfirmedOptOut.scala.html]: good spot, thank you 😃

File: app/views/optOut/ConfirmedOptOut.scala.html (hunk 2 above)
Comment (TomRafferty) [reply app/views/optOut/ConfirmedOptOut.scala.html:34]: Hi Pavel, as it wasn't something that was introduced in my work, I feel it's out of scope of the ticket (as it'll require a bit of refactoring and as you mentioned there are more occurrences of this too). So I'll get a ticket made up for tech debt 😃

**Review by Mbrookesy — APPROVED** [review]

**Pull request merged**